### ISSUE TITLE
2nd order methods?

### ISSUE THREAD

**Issue by GeoffNN** [openopt/copt#43]
Are second-order methods in the scope of the project?

**Comment by fabianp** [comment]
Sure

**Comment by fabianp** [comment]
But scipy.optimize already has some of these. Did you have any particular
method in mind?

**Comment by GeoffNN** [comment]
Right. I was thinking about other stochastic 2nd order methods. One question is: are we trying to have most state-of-the-art solvers implemented efficiently here, are we complementary to scipy.optimize ; in general, what are we aiming for?

**Comment by fabianp** [comment]
I would like a library that complements scipy.optimize, implementing methods that are not in that library without competing with it. 

That said, I propose to meet after NeurIPS deadline and discuss the project vision, as well as specific goals.
